tests/unit/test-cmd/test.run.js

```diff
@@ -326,101 +326,91 @@ describe('run', () => {
            sourceDir: '/fake/source/dir',
          },
        );
-     }
-  );
+     });
 
   describe('profile-create-new option', () => {
     beforeEach(() => {
       sinon.stub(fs, 'mkdir');
+      sinon.stub(fs, 'existsSync');
     });
 
     afterEach(() => {
       fs.mkdir.restore();
+      fs.existsSync.restore();
     });
 
-    it('creates dir when firefox profile doesn\'t exist',
-       async () => {
-         sinon.stub(fs, 'existsSync').returns(false);
-         const firefoxProfile = '/pretend/path/to/Firefox/profile';
-         const cmd = prepareRun();
-
-         await cmd.run({
-           firefoxProfile,
+    const fakeFirefoxProfile = '/pretend/path/to/Firefox/profile';
+    const fakeChromiumProfile = '/pretend/path/to/Chromium/profile';
+
+    async function testCreateProfileIfMissing(
+      existsSyncReturnValue,
```

Comment:
Nitpick: we may call this parameter as `expectProfileExists` and then use it for `fs.existsSync` and the if..else at line 355-359 (just because I prefer the name to rapresent the conditions of the tests instead of the name of the stubbed function that will return that value).